src/main/java/Storage.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Storage class to represent the tasks being stored into hard drive.
+ * Contains file location of saved task and the file f representing the saved tasks.
+ */
+public class Storage {
+
+    private String fileLocation = "./Data/Tasks.txt"; //hard-coded relative file location of stored tasks
```

Comment:
Relative location is dangerous because if you run your program in a different directory, your past data won't be found. This is especially so when you use jar.